include/deal.II/fe/mapping_fe_field.h

```diff
@@ -136,6 +136,20 @@ class MappingFEField : public Mapping<dim,spacedim>
   bool preserves_vertex_locations () const override;
 
 
+
+  /**
+   * Return the mapped vertices of a cell.
+   *
+   * This mapping ignores the vertices of the Triangulation it is associated to,
+   * and constructs the position of the vertices according to the @p euler_vector
+   * that was passed at construction time.
+   */
+  virtual
+  std::array<Point<spacedim>, GeometryInfo<dim>::vertices_per_cell>
+  get_vertices (const typename Triangulation<dim,spacedim>::cell_iterator &cell) const override;
+
```

Comment:
we usually keep just one space for headers

Reply:
yeah, i saw that the header is different to whitespaces elsewhere. Up to you, i would probably just fix it to be one space everywhere.